github.sgnl.yaml

```diff
@@ -0,0 +1,1099 @@
+        type: DateTime
+      - name: databaseId
+        externalId: databaseId
+        type: Int64
+      - name: description
+        externalId: description
+        type: String
+      - name: discussionsResourcePath
+        externalId: discussionsResourcePath
+        type: String
+      - name: discussionsUrl
+        externalId: discussionsUrl
+        type: String
+      - name: editTeamResourcePath
+        externalId: editTeamResourcePath
+        type: String
+      - name: editTeamUrl
+        externalId: editTeamUrl
+        type: String
+      - name: membersResourcePath
+        externalId: membersResourcePath
+        type: String
+      - name: membersUrl
+        externalId: membersUrl
+        type: String
+      - name: name
+        externalId: name
+        type: String
+      - name: newTeamResourcePath
+        externalId: newTeamResourcePath
+        type: String
+      - name: newTeamUrl
+        externalId: newTeamUrl
+        type: String
+      - name: repositoriesResourcePath
+        externalId: repositoriesResourcePath
+        type: String
+      - name: repositoriesUrl
+        externalId: repositoriesUrl
+        type: String
+      - name: resourcePath
+        externalId: resourcePath
+        type: String
+      - name: slug
+        externalId: slug
+        type: String
+      - name: teamsResourcePath
+        externalId: teamsResourcePath
+        type: String
+      - name: teamsUrl
+        externalId: teamsUrl
+        type: String
+      - name: updatedAt
+        externalId: updatedAt
+        type: DateTime
+      - name: url
+        externalId: url
+        type: String
+      - name: viewerCanAdminister 
+        externalId: viewerCanAdminister
+        type: Bool
+      - name: viewerCanSubscribe
+        externalId: viewerCanSubscribe
+        type: Bool
+  # Fetched from the 'members' attribute belonging to the Teams entity.
+  # TeamMember entity is for creating relationships between Users and Teams and does not support organizationVerifiedDomainEmails.
+  # To retrieve organizationVerifiedDomainEmails, declare it as an attribute for the User entity.
+  # TeamMember is a User entity type: https://docs.github.com/en/graphql/reference/objects#user
+  TeamMember:
+    parent: Team
+    displayName: TeamMember
+    externalId: $.members.edges
+    description: Entity representing a GitHub team each user belongs to.
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: $.node.id
+        type: String
+        indexed: true
+        uniqueId: true
+        relationshipAttributeId: TeamMember-id
+      # Note: Role represents the TeamMember's role on the team.
+      # https://docs.github.com/en/graphql/reference/enums#teammemberrole
+      - name: role
+        externalId: role
+        type: String
+      - name: avatarUrl
```

Comment:
Apart from `id/login/email` (a unique identifier for a User) and `role` (in the context of a Team), if there's no attributes unique to a team-member as compared to a user - then we can exclude these attributes altogether.

Team member would be same-as User right?
User would be a superset of Team member right?

Reply:
fixed